packages/lib/services/database/migrations/47.ts

```diff
@@ -0,0 +1,7 @@
+import { SqlQuery } from '../types';
+
+export default (): (SqlQuery|string)[] => {
+	return [
+		'ALTER TABLE sync_items ADD COLUMN sync_warning_dismissed INT NOT NULL DEFAULT "0"',
```

Comment:
You use the term "ignored" in some places and "dismissed" in other places. Would it be possible to use the same term everywhere?